wp-content/themes/aerospace/package.json

```diff
@@ -57,6 +57,7 @@
     "eslint-fix": "gulp eslint --fix"
   },
   "dependencies": {
-    "core-js": "^3.6.5"
+    "core-js": "^3.6.5",
+    "typescript": "^4.2.3"
```

Comment:
Why did you add `typescript` as a dependency? This project doesn't use typescript.